src/fhirtypes/common.ts

```diff
@@ -340,6 +340,7 @@ export function listUndefinedLocalCodes(
 ): void {
   let undefinedCodes: string[] = [];
   applyInsertRules(codeSystem, tank);
+  const conceptRulePath = /^(concept(\[(\d+|\+|=)\])?\.)+code$/;
```

Comment:
The grammar is a lot more lenient about whitespace than this regex is.  Maybe we should at least allow for variable whitespace inside the `[]`?  Just in case someone tries to align things like this:
```
* concept[ 8].code = #eight
* concept[ 9].code = #nine
* concept[10].code = #ten
```

Reply:
Ha.  Now I see that you merely just moved this regex up.  But you were the last to touch it now.  So now it's your responsibility! ;-)